api/debug_handler.go

```diff
@@ -725,3 +728,18 @@ func (handler *DebugHandler) ReadBackfillQueueUpsertBatch(w http.ResponseWriter,
 	common.RespondWithJSONObject(w, response)
 	return
 }
+
+// Bootstrap will turn on bootstrap based on the request.
+func (handler *DebugHandler) BootstrapRetry(w http.ResponseWriter, r *http.Request) {
```

Comment:
need to prevent bootstrap again if bootstrap is running or submitted

Reply:
it is protected in bootstrap manager.